src/app/component/Select/Select.style.scss

```diff
@@ -8,7 +8,20 @@
  * @package scandipwa/base-theme
  * @link https://github.com/scandipwa/base-theme
  */
- .Select {
+
```

Comment:
While here, if not already done, please adjust the arrow position and rotation to be centered if rotated.